src/App.js

```diff
@@ -2,13 +2,111 @@ import React, { Component } from 'react';
 import './App.scss';
 
 export default class App extends Component {
+
+  // setting initial state to match the schema on https://jsonresume.org/schema/
+  state = {
+    data: {
+      basics: {
+        name: null,
+        label: null,
+        picture: null,
+        email: null,
+        phone: null,
+        website: null,
+        summary: null,
+        location: {
+          address: null,
+          postalCode: null,
+          city: null,
+          countryCode: null,
+          region: null
+        },
+        profiles: []
+      },
+      work: [],
+      volunteer: [],
+      education: [],
+      awards: [],
+      publications: [],
+      skills: [],
+      languages: [],
+      interests: [],
+      references: []
+    }
+  }
+
+  // adds a new entry to the respective section
+  addNew = (event) => {
+    const entryArray = this.state.data[event.target.value].slice()
+    if (event.target.value === "education") {
```

Comment:
So you would do something like this for the other sections right? awards, volunteer, publications skills.

Reply:
Exactly, there were a lot more fields and I didn't manage my time correctly so I couldn't affect the rest of the fields.